### ISSUE TITLE
Branches moving forward

### ISSUE THREAD

**Issue by lauxjpn** [PomeloFoundation/Pomelo.EntityFrameworkCore.MySql#862]
For applying maintenance bug fixes to 2.2.6, we should create a corresponding `2.2.6` branch.

I think we should also merge the `3.0.0-wip` branch into `master` now and create a `3.0.0` branch at the latest, when the GA 3.0.0 release is out and we are starting the work to target EF Core 3.1.

**Comment by lauxjpn** [comment]
@caleblloyd Is that in line with our development process and CI model?

**Comment by caleblloyd** [comment]
Yes, that's a good idea, I made a `2.2-maint` branch that can serve as the 2.2.x maintenance branch

Once EF Upstream is stable on a minor version, that can align with `master` and we can move the old minor version to a `-maint` branch

I'll also add continuous nightlies on `-maint` branches

Do you think 2.2.6 RTM should be published?  RC2 has been out for 8 days but only downloaded 82 times on NuGet.  I don't think we've received any reports of regressions though?

**Comment by lauxjpn** [comment]
I think we can publish 2.2.6 as RTM. I am not aware of any 2.2.6 related issues or regressions.

**Comment by caleblloyd** [comment]
2.2.6 published from the maintenance branch. `master` should be ready to merge into, resolved merge conflicts last night

**Comment by lauxjpn** [comment]
Thanks for the work! I will merge `2.2.6` and `3.0.0-wip` into `master` today.
